www/public/site.css

```diff
@@ -50,10 +50,8 @@ html {
 
 html,
 body {
-    margin: 0;
+    margin: 0 30px;
     padding: 0;
-    width: 100%;
-    height: 100%;
```

Comment:
Hm, it looks like this is the global style - can we revise it to be scoped just to that page? (e.g. with an id selector or something.)

This might be fine, but to accept any site-wide CSS change like this I'd have to manually QA every page on the site in all browsers and mobile devices, and I really don't want to do that for a bug that's only affecting one page. 😅